releases/0.2.11/CHANGELOG_EXCERPT.md

```
@@ -1,6 +1,6 @@
-# EVOSEAL 0.2.5 - Release Notes
+# EVOSEAL 0.2.11 - Release Notes
 
-*Released on: 2025-07-22*
+*Released on: 2025-07-23*
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)